.circleci/config.yml

```diff
@@ -583,7 +583,35 @@ jobs:
         - image: cimg/python:3.10
           environment:
             GETH_VERSION: v1.10.18
-            TOXENV: py310-install-geth-v1.10.18
```

Comment:
It looks like lines 581-586 all have an extra indent from my last commit. I think I added this manually since I added python 3.10 support, though I probably should've ran the script after instead 😅.

Could you just add that into this PR so it's evened out? I'm not sure that it affected the config itself but I think it might now that line 586 was pulled back a tab. Either way, would just be nice to have it all formatted correctly again.

Reply:
that was the single test that was failing, so maybe was very important. Thanks for catching!